docs/content/install/install-on-google-cloud-run.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Install on Google Cloud Run"
+date: 2019-10-10T18:16:43+11:00
+draft: false
+weight: 4
+---
+
+[Google Cloud Run](https://cloud.google.com/run/) is a service that aims to bridge the gap between the maintainance benefits of serverless architecture and the flexibility of Kubernetes. It is built on top of the opensource [Knative](https://knative.dev/) project. Deploying using Cloud Run is similar to deploying using [Google App Engine](/install/install-on-gae) with the benefits of a free tier and a simpler build process. 
+
+## Selecting a Storage Provider
+
+There is documentaion about how to use environment variables to configure a large number of storage providers; however, for this prarticular example we will use [Google Cloud Storage](https://cloud.google.com/storage/)(GCS) because it fits nicely with Cloud Run.
+
+## Before You Begin
+
+This guide assumes you have completed the following tasks:
+
+- Signed up for Google Cloud
+- Installed the [gcloud](https://cloud.google.com/sdk/install) command line tool 
+- Installed the beta plugin for ghe gcloud command line tool ([this is how to set it up](https://cloud.google.com/run/docs/setup))
+- Created a (GCS) bucket for your go modules
+
+## Setup
+
+First set up configuration environment variables
+
+```bash
+export GOOGLE_CLOUD_PROJECT=your-project
+export ATHENS_REGION=asia-northeast1
+export GCS_BUCKET=your-bucket-name
+```
```

Comment:
Would you mind making it clear here that the user can change `your-project`, `asia-northeast1`, and `your-bucket-name` to values that make sense for them. What do you think of the below text?

"Change the values of these environment variables to be appropriate for your application. For `GOOGLE_CLOUD_PROJECT`, this needs to be the name of the project that has your cloud run deployment in it. `ATHENS_REGION` should be the region that your cloud run instance will be in (_maybe a link here to the regions list?_), and `GCS_BUCKET` should be the Google Cloud Storage bucket that Athens will store module code and metadata in."

Reply:
I think that is much clearer. I added the list of regions to this page as a result of the previous comment, so I didn't repeat the link. Is that alright?

Reply:
Yes, this is much better.